tests/foreman/api/test_multiple_paths.py

```diff
@@ -247,52 +205,31 @@ def test_post_status_code(self, entity_cls):
         @Assert: HTTP 201 is returned with an ``application/json`` content-type
 
         """
-        logger.debug('test_post_status_code arg: %s', entity_cls)
-        skip_if_sam(self, entity_cls)
-
-        # Libvirt compute resources suffer from BZ 1118015. However, partials
-        # cannot be compared for class identity and the class hierarchy needs
-        # fixing (SatelliteQE/nailgun#42), so we just comment it out above.
-        if entity_cls in BZ_1118015_ENTITIES and bz_bug_is_open(1118015):
-            self.skipTest('Bugzilla bug 1118015 is open.')
-
-        response = entity_cls().create_raw()
-        self.assertEqual(httplib.CREATED, response.status_code)
-        self.assertIn('application/json', response.headers['content-type'])
+        exclude_list = (
+            entities.TemplateKind,  # see comments in class definition
+        )
+        for entity_cls in set(valid_entities()) - set(exclude_list):
+            with self.subTest(entity_cls):
+                logger.debug('test_post_status_code arg: %s', entity_cls)
+                skip_if_sam(self, entity_cls)
+
+                # Libvirt compute resources suffer from BZ 1118015. However,
+                # partials cannot be compared for class identity and the class
+                # hierarchy needs fixing (SatelliteQE/nailgun#42), so we just
+                # comment it out above.
+                if (entity_cls in BZ_1118015_ENTITIES and
+                        bz_bug_is_open(1118015)):
+                    self.skipTest('Bugzilla bug 1118015 is open.')
```

Comment:
I'm curious to know what happens if this condition is true? Will the rest of the test be skipped once it is true?